src/Clapi/Valuespace.hs

```diff
@@ -50,24 +50,21 @@ fromRight (Right b) = b
 mapFilterJust :: Map.Map k (Maybe a) -> Map.Map k a
 mapFilterJust = fmap fromJust . Map.filter isJust
 
-data Liberty = Cannot | May | Must deriving (Show, Eq, Enum, Bounded)
+data Liberty = Cannot | May | Must deriving (Show, Eq, Ord, Enum, Bounded)
```

Comment:
Don't think this needs to be orderable. @foolswood says this was because we were playing around with it being in a set.